examples/ecs/query_bundle.rs

```diff
@@ -0,0 +1,69 @@
+use bevy::{
+    // diagnostic::LogDiagnosticsPlugin,
+    ecs::schedule::RunOnce,
+    log::{LogPlugin, LogSettings},
+    prelude::*,
+};
+
+fn main() {
+    App::build()
+        .insert_resource(LogSettings {
+            level: bevy::log::Level::DEBUG,
+            ..Default::default()
+        })
+        .add_plugin(LogPlugin)
+        .add_plugins(MinimalPlugins)
+        .insert_resource(bevy::app::ScheduleRunnerSettings::run_once())
+        .add_startup_system(setup.system())
+        .add_stage("diagnostic", SystemStage::single_threaded())
+        .add_system_to_stage(
+            "diagnostic",
+            query_component_without_bundle
+                .system()
+                .with_run_criteria(RunOnce::default()),
+        )
+        .add_system_to_stage(
+            "diagnostic",
+            test_query_bundle
+                .system()
+                .with_run_criteria(RunOnce::default()),
+        )
+        .run();
+}
+
+#[derive(Debug)]
+struct Dummy(usize);
```

Comment:
Maybe use something a bit less abstract here? Maybe `Name`, `Age`, and `PersonBundle`?